doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -182,51 +180,44 @@ You can download latest Service OS kernel from
                 linux  /boot/acrn/org.clearlinux.iot-lts2018-sos.4.19.78-98  pci_devices_ignore=(0:18:1) console=tty0 console=ttyS0 root=PARTUUID=<UUID of rootfs partition> rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_gvt=1 i915.enable_guc=0 hvlog=2M@0x1FE00000
         }
 
-   .. note::
-        You need to adjust this to use your partition UUID (``PARTUUID``) for
-        the ``root=`` parameter (or use the device node directly).
+     .. note::
+          Adjust this to use your partition UUID (``PARTUUID``) for the ``root=`` parameter (or use the device node directly).
```

Comment:
The alignment seems off here, or is it me?

Reply:
@gvancuts I made a few more updates and they aligned correctly when I viewed them locally. I'll check again.